src/tools/sample/sample.cpp

```diff
@@ -15,6 +15,25 @@
     direction. The connection only shuts down when the 1 second idle timeout
     triggers.
 
+    A certificate needs to be available for the server to function.
+
+    On windows, the following powershell command can be used to generate a self
+    signed certificate with the correct settings. This works for both Schannel
+    and OpenSSL TLS providers, assuming the KeyExportPolicy parameter is set to
+    Exportable. The Thumbprint received from the command is then passed to this
+    sample with -cert_hash:PASTE_THE_THUMBPRINT_HERE
+
+    New-SelfSignedCertificate -DnsName $env:computername,localhost -FriendlyName MsQuic-Test -KeyUsageProperty Sign -KeyUsage DigitalSignature -CertStoreLocation cert:\CurrentUser\My -HashAlgorithm SHA256 -Provider "Microsoft Software Key Storage Provider" -KeyExportPolicy Exportable
+
+    On linux, the following command can be used to generate a self signed
+    certificate that works with the OpenSSL TLS Provider. This can also be used
+    for Windows OpenSSL, however we recommend the cert store method above for
```

Comment:
```suggestion
    for Windows OpenSSL, however we recommend the certificate store method above for
```
and going to have to fix the too long of line...